### PR TITLE
Add Dependency react-router-dom

### PR DESCRIPTION
**Pull request by realesh**
add dependency for react-router-dom

### CODE REVIEW

**Comment by LostPrime** [comment]
LGTM! Ship It!

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "react": "^16.7.0",
     "react-dom": "^16.7.0",
+    "react-router-dom": "^4.3.1",
```
Comment (greensirius) [inline package.json:8]: Is that the last version of rrd?

**Review by greensirius — APPROVED** [review]
Looks Got To Me.
Ship it

**Review by LostPrime — APPROVED** [review]
LGTM again, Ship it!

**Pull request merged**